### PR TITLE
test(hx-select): surf-2024

### PR DESCRIPTION
**Pull request by BY00565233**
## Description

unit test `<hx-select>`

image

### What are the relevant story cards/tickets? Any additional PRs or other references?

Jira: SURF-2024(unit test case)

## Before you request a review for this PR:

- [x] Did you add component tests for any new code?
- [x] Did you run the component unit tests via `yarn test` to ensure all tests passed?
- [x] Did you include a screenshot of the component tests?
- [x] If you changed/added functionality, did you update the demo page and documentation?
- [x] If needed, did you add or modify the demo test page to test the changed/added functionality?
- [x] Did you assign reviewers?
- [x] In Jira, have you linked to this PR on the ticket(s)?

### CODE REVIEW

File: src/elements/hx-select/index.spec.js [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+                const shadow = component.shadowRoot.innerHTML;
+
+                expect(component).shadowDom.to.equal(shadow);
+            });
+
+            it('should render the Shadow Root mode open', async () => {
+                const component = /** @type { HXSelectElement } */ await fixture(template);
+                const mode = component.shadowRoot.mode;
+
+                expect(mode).to.equal("open");
```
Comment (100stacks) [inline src/elements/hx-select/index.spec.js:48]: Please use single quotes here - `'open'`:

File: src/elements/hx-select/index.spec.js [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+                const id = 'hxSelect';
+                const component = /** @type { HXSelectElement } */ await fixture(template);
+                const shadow = component.shadowRoot;
+                const query = shadow.querySelector(elSelector);
+                const queryId = query.id;
+
+                expect(queryId).to.equal(id);
+            });
+
+            it('markup should contain a #hxTrigger', async () => {
```
Comment (100stacks) [inline src/elements/hx-select/index.spec.js:64]: Please add `<div>` to test description:

```js
it('markup should contain a #hxTrigger <div>', async () => {
```

File: src/elements/hx-select/index.spec.js (hunk 1 above)
Comment (BY00565233) [reply src/elements/hx-select/index.spec.js]: Changes have been done, Please review

File: src/elements/hx-select/index.spec.js (hunk 2 above)
Comment (BY00565233) [reply src/elements/hx-select/index.spec.js]: Changes have been done, Please review

**Pull request merged**